.ddev/config.yaml

```diff
@@ -0,0 +1,195 @@
+name: symfony-keycloak-bundle
```

Comment:
Why do a symfony bundle need a .ddev config?

Reply:
we assume that we do not have local php setups, otherwise it´s not possible to run tests